docs/wiki/archive/Magnetometer.mdx

```diff
@@ -17,56 +20,65 @@ Mag information is essential for position hold, which we intend to support.
 
+:::caution
 The orientation of the magnetometer on the quad is very important. In Betaflight, the data from the magnetometer must be provided as follows:
 
-- X forward
-- Y left
-- Z up
+- X points **forward**
+- Y points **left**
+- Z points **up**
+:::
 
-![Mag axes and orientation](../images/MagOrientationDiagram.png)
-Fig 1 - Mag axes and orientation
+<figure className="align-center">
+  <img src={MagOrientation} alt="Figure of magnetometer axes and orientation" className="no-effect " />
+  <figcaption>
+    <b>Fig. 1</b> - Magnetometer axes and orientation in relation to the drone as expected by Betaflight
+  </figcaption>
+  <br />
+</figure>
 
-If the magnetometer cannot be physically positioned so that it's axes are aligned as per the diagram above, the `Mag alignment` setting in Configurator (`align_mag` in CLI) may be used to correct for unusual Mag orientations. For example, some GPS units mount the mag unit upside down.
+If the magnetometer cannot be physically positioned so that its axes are aligned as per the diagram above, the `Mag alignment` setting in Configurator (`align_mag` in the [CLI](../configurator/cli-tab)) may be used to correct for other mag orientations. For example, some GPS units mount the mag unit upside down and some mag models use an unusual axis orientation.
 
 Correct orientation for a given axis can be confirmed by checking the Magnetometer data in the Sensors tab.
 
-## About the earth's magnetic field vector
+## About the Earth's Magnetic Vector Field
 
-The magnetic field of the earth doesn't simply point North and South, and is not parallel to the ground.
+[Earth's magnetic field](https://en.wikipedia.org/wiki/Earth%27s_magnetic_field)[^1] is not a uniform vector field. It is not simply oriented to the geographic North or South, and is not parallel to the ground in most places around the world. In other words: The field direction and field strength both vary from place to place.
 
-It is a vector that usually points a few degrees away from true North, and points either down into the ground in the Northern Hemisphere, or up out of the ground in the Southern Hemisphere. There are considerable angular deviations, depending upon where you are on the surface of the Earth.
+Earth's magnetic field usually gets represented in a **NED** (north-east-down) frame of reference. This means that the X axis points North, the Y axis points East, and the Z axis points Down. This is not at all relevant for Betaflight's mag orientation on the quad, but it's useful to know for interpreting the magnetic field at your location and for navigating around the earth's surface.
 
```

Comment:
The Earth's magnetic field is not described in terms of X, Y or Z.  It's described in terms of North, South, East, West, and an inclination angle with respect to the horizon.

X, Y and Z are axes that relate to the sensor, not to the the earth, and the sensor can be at any orientation with respect to the earth.  Hence either of the X, Y and Z axes could point to any angle on the earth.  

We don't want to confuse our users by saying that 'X relates to North' because in a a practical sense, all that matters to the user is that the X axis should point to the nose of the quad (or that the data be transformed so that the data returned matches that orientation).  Whether the X axis of the sensor points North or South depends which way the quad is flying at the time and has nothing do with how it needs to be oriented on the quad.

While the orientation of an aircraft is often drawn as X forward, Y right, and Z down, that's not the orientation of the mag that we need for Betaflight.

I don't see a lot of value in discussing NED, ENU, or other complicated ways of thinking which attempt to link a max X axis with a specific compass orientation.  Doing that will only confuse people.  We could add a side-bar, or a note, right at the bottom of the document, that discusses craft or field orientation models of those kinds, and explain and how Betaflight's mag orientation defaults contrast with those models.  But at this point in the document it will only confuse people.